src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearchindexer/health/IndexInfo.kt

```diff
@@ -6,16 +6,16 @@ import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.model.SyncIndex.BLUE
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.model.SyncIndex.GREEN
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.repository.PrisonerRepository
-import uk.gov.justice.digital.hmpps.prisonersearchindexer.services.IndexService
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.services.IndexStatusService
+import uk.gov.justice.digital.hmpps.prisonersearchindexer.services.PopulateIndexService
 import uk.gov.justice.hmpps.sqs.HmppsQueue
 import uk.gov.justice.hmpps.sqs.HmppsQueueService
 import uk.gov.justice.hmpps.sqs.countMessagesOnQueue
 
 @Component
 class IndexInfo(
   private val indexStatusService: IndexStatusService,
-  private val indexService: IndexService,
+  private val populateIndexService: PopulateIndexService,
```

Comment:
this isn't quite right - need to pull out the `getIndexCount` into a separate service, but will do in next PR